### PR TITLE
Use Fira fonts v4.3

### PR DESCRIPTION
**Pull request by joe-skb7**
Use newer version of Fira fonts from here:

    https://github.com/bBoxType/FiraSans

Signed-off-by: Sam Protsenko <joe.skb7@gmail.com>

### CODE REVIEW

**Comment by joe-skb7** [comment]
Of course, some changes should be made to debian/* files. This patch is more like a hack. Btw, have you thought about pushing this package to official Debian repo? These fonts seem quite popular in presentations (e.g. in Latex + Beamer + Metropolis).

File: Makefile [hunk 1]
```diff
@@ -16,27 +16,16 @@
...
 
-
-## check with https://github.com/adobe-fonts/source-code-pro/releases for version
-srcfile1=https://github.com/mozilla/Fira/archive/4.202.tar.gz
-outfile1=$(shell basename ${srcfile1})
-srcfile2=https://github.com/mozilla/Fira/archive/4.106.tar.gz
-outfile2=$(shell basename ${srcfile2})
-
+ARCHIVE		= master.zip
+ARCHIVE_LINK	= https://github.com/bBoxType/FiraSans/archive/$(ARCHIVE)
```
Comment (eddelbuettel) [inline Makefile:20]: What is this URL? 

I was careful to download from Mozilla having read their copyright.  Can we not just use a newer file from them?

File: Makefile [hunk 2]
```diff
@@ -16,27 +16,16 @@
...
-	mv Fira-*/otf/* otf/
-	mv Fira-*/ttf/* ttf/
-	rm -rf Fira-* 
-	test -f ${outfile2} || wget ${srcfile2} #
-	tar xfz ${outfile2}
-	mv Fira-*/otf/*  otf/
-	mv Fira-*/ttf/*  ttf/
-	rm -rf Fira-* 
+	find ${UNPACKED_DIR} -name '*.otf' -exec cp {} otf/ \;
+	find ${UNPACKED_DIR} -name '*.ttf' -exec cp {} ttf/ \;
```
Comment (eddelbuettel) [inline Makefile:28]: Your changes are elegant and clean, yet maybe I would rather just stick with what obviously worked before ...

**Comment by eddelbuettel** [comment]
I alrady have [100+ Debian packages to look after](https://qa.debian.org/developer.php?login=edd@debian.org) so at the time I decided I'd rather not take on more. 

And yes, I do use this with mtheme/Metropolis myself.  Which is now in texlive so as I recall, the font _is_ now in Debian.  And [this package search](https://packages.debian.org/search?searchon=contents&keywords=Fira&mode=filename&suite=unstable&arch=any) seems to agree.

File: Makefile (hunk 1 above)
Comment (eddelbuettel) [reply Makefile:20]: Never mind -- looked there now.

**Comment by eddelbuettel** [comment]
Thanks for the PR. Much appreciated!

**Pull request merged**